qiskit/synthesis/evolution/lie_trotter.py

```diff
@@ -60,6 +60,7 @@ def __init__(
             | None
         ) = None,
         wrap: bool = False,
+        preserve_order: bool = False,
```

Comment:
I thought the consensus here was to not reorder terms by default because of the potential performance overhead. Shouldn't the default be to `True`?